app/views/planning_applications/_planning_application_table.html.erb

```diff
@@ -7,14 +7,11 @@
       <th scope="col" class="govuk-table__header">Site address</th>
       <th scope="col" class="govuk-table__header">Application type</th>
       <th scope="col" class="govuk-table__header">Target date</th>
-      <% if id == "closed" %>
-        <th scope="col" class="govuk-table__header">Decision</th>
-      <% else %>
+      <% if id != "closed" %>
         <th scope="col" class="govuk-table__header">Days left</th>
       <% end %>
-      <% if (id == "in_assessment" || id == "not_started_and_invalid") %>
         <th scope="col" class="govuk-table__header">Status</th>
```

Comment:
This need to be outdented by 1 level, as it looks like it's within the if statement to me